tools/wpt/run.py

```diff
@@ -180,11 +180,10 @@ def install(self, channel=None):
         if self.prompt_install(self.name):
             return self.browser.install(self.venv.path, channel)
 
-    def install_requirements(self):
+    def requirements(self):
         if not self.venv.skip_virtualenv_setup and self.browser.requirements:
```

Comment:
How about dropping the first condition here, since it's checked at the call site anyway?